lightning/src/ln/msgs.rs

```diff
@@ -66,16 +66,14 @@ pub enum DecodeError {
 }
 
 /// An init message to be sent or received from a peer
-#[derive(Clone)]
+#[derive(Clone, Debug, PartialEq)]
 pub struct Init {
-	#[cfg(not(feature = "fuzztarget"))]
-	pub(crate) features: InitFeatures,
-	#[cfg(feature = "fuzztarget")]
+	/// The relevant features which the sender supports
 	pub features: InitFeatures,
```

Comment:
I take it this was necessary for the bindings?

Reply:
I'm honestly not sure why it was here, I think it's just left over from when we had all the message guts hidden instead of exposed. Features is a well-exposed type now, though.